src/modules/facts/reducer.js

```diff
@@ -18,15 +18,35 @@ export function factsR(state = INITIAL_STATE, action) {
       }
     case 'ADD_FACT_TO_ORG': {
       const oldOrganizationFact = state.organizationFacts.find(e => e.variable_name === action.organizationVariableName)
-      const oldChildren = !!oldOrganizationFact ? oldOrganizationFact.children.filter(c => c.variable_name !== action.fact.variable_name) : []
+      return {
+        ...state,
+        organizationFacts: [
+          { variable_name: action.organizationVariableName, children: [...(oldOrganizationFact.children || []), action.fact] },
+          ...state.organizationFacts.filter(e => e.variable_name !== action.organizationVariableName)
```

Comment:
then have this not reference ``action.organizationVariableName``, but instead ``newOrganizationfact.organizationVariableName``